ugvc/mrd/mrd_utils.py

```diff
@@ -301,7 +313,7 @@ def read_signature(  # pylint: disable=too-many-branches
                 for variant_file in f_bw:
                     variant_file.close()
         df_sig = pd.concat(df_list).astype({"coverage": int})
-
+    df_sig["hmer"] = np.nan  # it read TRUE/FALSE in some cases, so we reset it to nan
```

Comment:
1. Please explain
2. I don't like this patch, if the issue is that sometime you get string values then find a way to force-convert to float (as in, convert to float if possible and it not set to NaN), pretty sure pandas can do that